src/jinja2_fragments/sanic.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Dict, Optional
+
+from jinja2 import Environment
+
+try:
+    from sanic import Request, Sanic, SanicException
+    from sanic_ext.exceptions import ExtensionNotFound
+    from sanic_ext.extensions.templating import render as sanic_render
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "Install sanic, sanic_ext before using jinja2_fragments.sanic"
+    ) from e
+
+from . import render_block, render_block_async
+
+
+async def render(
+    template_name: str = "",
+    status: int = 200,
+    headers: Optional[Dict[str, str]] = None,
+    content_type: str = "text/html; charset=utf-8",
+    app: Optional[Sanic] = None,
+    environment: Optional[Environment] = None,
+    context: Optional[Dict[str, Any]] = None,
+    *,
+    block: Optional[str] = None
+) -> sanic_render.TemplateResponse:
+    if block is None or block == "":
+        return await sanic_render.render(
+            template_name=template_name,
+            status=status,
+            headers=headers,
+            content_type=content_type,
+            app=app,
+            environment=environment,
+            context=context,
+        )
+    if app is None:
+        try:
+            app = Sanic.get_app()
+        except SanicException as e:
+            raise SanicException(
+                "Cannot render template beause locating the Sanic application "
```

Comment:
Typo on "because". I saw this same type is on the original sanic_ext source code also :D

```suggestion
                "Cannot render template because locating the Sanic application "
```